stock_picking_dispatch/wizard/dispatch_start.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# © 2012-2014 Guewen Baconnier, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from datetime import date
+from openerp import _, api, fields, models
+
+
+class PickingDispatchStart(models.TransientModel):
+    _name = 'picking.dispatch.start'
+    _description = 'Picking Dispatch Start'
+
+    def _get_message(self):
+        selected_count = len(self.env.context['active_ids'])
+
+        assigned_count = self.env['picking.dispatch'].search_count(
+            self._get_assigned_domain()
+        )
+
+        message = "<ul><li>%s</li>" % _(
+            "%s dispatch(es) will be started</li>"
```

Comment:
Indeed, I will remove it but I was a little bit surprised because tests passed and there is no double </li> in test assertion!!

Finally I found that HTML field sanitize his content and remove extra tags

https://github.com/OCA/OCB/blob/9.0/openerp/osv/fields.py#L343
https://github.com/OCA/OCB/blob/9.0/openerp/tools/mail.py#L98